Test/git-issues/git-issue-1963a.dfy

```diff
@@ -9,7 +9,7 @@ module A {
 }
 
 module B refines A {
-  newtype NT = ... x: int | x < 0 witness -1 // Error: Change (or repeat) a newtype body
+  newtype NT = ... | x < 0 witness -1 // Error: Change (or repeat) a newtype body
```

Comment:
What would be the error triggered by the previous one now?

Reply:
Now: Error: in refining a newtype, the '...' replaces the '=' token and everything up to the left brace starting the declaration of the newtype body (if any); a newtype refinement may not change the base type of the newtype